controller/workitem.go

```diff
@@ -679,6 +741,30 @@ func ConvertWorkItemsToCSV(ctx context.Context, app application.Application, all
 	return buf.String(), columnLabels, nil
 }
 
+// resolveNumberByWorkItemID retrieves the work item number for a work item ID
+func resolveNumberByWorkItemID(ctx context.Context, db application.DB, workItemID uuid.UUID) (string, error) {
+	var numberInt int
+	err := application.Transactional(db, func(appl application.Application) error {
+		var err error
+		thisWorkItem, err := appl.WorkItems().LoadByID(ctx, workItemID)
+		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"err":        err,
+				"workItemID": workItemID,
+			}, "unable to retrieve space")
+			return errs.Wrapf(err, "error retrieving number for workItemID: %s", workItemID.String())
+		}
+		number := (*thisWorkItem).Fields[workitem.SystemNumber]
```

Comment:
You don't need to deference the pointer in golang.
Take a look at 
https://github.com/fabric8-services/fabric8-wit/blob/ce0056800674ca481a55e60645200c7e4c266884/controller/work_item_labels.go#L38-L43

Reply:
Fixed 4fb718e.